dotcom-rendering/src/web/components/Card/components/ImageWrapper.tsx

```diff
@@ -58,9 +60,15 @@ export const ImageWrapper = ({
 		<div
 			css={[
 				isHorizontal &&
+					imageType === 'mainmedia' &&
 					flexBasisStyles({
 						imageSize,
 					}),
```

Comment:
Very much styleistic but could the order be 
```js
imageType === 'mainmedia' &&
isHorizontal &&
flexBasisStyles({
	imageSize,
}),
```

Reply:
Yeah, this looks a bit nicer. Done 👍